SoftLayer/CLI/firewall/add.py

```diff
@@ -15,7 +15,7 @@
               type=click.Choice(['vs', 'vlan', 'server']),
               help='Firewall type',
               required=True)
-@click.option('--high-availability', '--ha',
+@click.option('--high-availability', '-ha',
```

Comment:
Instead of changing the options, you can just switch the order of "--high-availability" and "--ha". The non-flag is used as the keyword argument. Flags aren't usually more than one character long because you can use them like this: `command -a -b -c`and this: `command -abc`